web/src/constants/server-error.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const SERVER_ERROR = 'Oops, something is wrong. Please try again later!';
```

Comment:
Are there going to be more server error messages put in this file? I would suggest to rename this single error constant to "GENERIC_ERROR".

Reply:
okay!